data-prepper-api/src/main/java/com/amazon/dataprepper/model/event/JacksonEvent.java

```diff
@@ -76,6 +76,12 @@ protected JacksonEvent(final Builder builder) {
         this.jsonNode = getInitialJsonNode(builder.data);
     }
 
+    static Event fromMessage(String message) {
+        return JacksonEvent.builder()
```

Comment:
I think this should look like:

```
return Event event = JacksonEvent
    .builder()
    .withEventType("event")
    .withData(Collections.singletonMap("message", message)
    .build();
```

You will need to change the unit test accordingly.

Reply:
Sure, I can add `withData()` in the builder, I only added `eventType` since it's a required field.

Reply:
The event shouldn't be random. From what I can tell of our current usage, this can be just the literal `"event"`.

It also might be good to extract both literals ("event" and "message") into constants.